postman_example/Enable Banking.postman_collection.json

```diff
@@ -210,6 +211,211 @@
 				}
 			},
 			"response": []
+		},
+		{
+			"name": "Starting payment",
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Authorization",
+						"value": "Bearer {{jwt_signed}}",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"aspsp\": {\n        \"name\": \"Nordea\",\n        \"country\": \"FI\"\n    },\n    \"state\": \"b463a960-9616-4df6-909f-f80884190c22\",\n    \"redirect_url\": \"https://google.com/\",\n    \"psu_type\": \"personal\",\n    \"payment_type\": \"SEPA\",\n    \"payment_request\": {\n        \"credit_transfer_transaction\": [\n            {\n                \"instructed_amount\": {\n                    \"amount\": \"1.00\",\n                    \"currency\": \"EUR\"\n                },\n                \"beneficiary\": {\n                    \"creditor\": {\n                        \"name\": \"Tester\"\n                    },\n                    \"creditor_account\": {\n                        \"scheme_name\": \"IBAN\",\n                        \"identification\": \"FI2112345600000785\"\n                    }\n                },\n                \"remittance_information\": [\n                    \"testing\"\n                ]\n            }\n        ]\n    }\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "https://api.enablebanking.com/payments",
+					"protocol": "https",
+					"host": [
+						"api",
+						"enablebanking",
+						"com"
+					],
+					"path": [
+						"payments"
+					]
+				}
+			},
+			"response": [
+				{
+					"name": "HandelsBanken (sandbox)",
```

Comment:
I'd name it

SE Handelsbanken DOMESTIC personal

i.e. {country} {ASPSP name} {payment type} {psu type}

we could also add auth method name at the end, but not sure if it's absolutely necessary